src/plugins/intel_cpu/src/nodes/input.h

```diff
@@ -13,16 +13,47 @@ namespace node {
 
 class Input : public Node {
 public:
+    struct InputConfig {
+        MemoryDescPtr desc;
+        bool inPlace;
+    };
```

Comment:
Can we reuse the port config for this purpose just to avoid additional entities?

Reply:
I want to do so initially.
The only piece we miss is an "any" descriptor
Then for an OutputConfig we could also use PortConfig

Reply:
Ok then leave the `InputConfig` struct till `any` desc is introduced.